### PR TITLE
Support for Ufe cameras.

### PR DESCRIPTION
**Pull request by williamkrick**

### CODE REVIEW

File: lib/mayaUsd/ufe/UsdCameraHandler.h [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+#pragma once
+
+#include <mayaUsd/base/api.h>
+#include <mayaUsd/ufe/UsdCamera.h>
```
Comment (ppt-adsk) [inline lib/mayaUsd/ufe/UsdCameraHandler.h:19]: Is this one needed?

File: lib/mayaUsd/ufe/UsdCameraHandler.h [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+#include <ufe/cameraHandler.h>
+
+namespace MAYAUSD_NS_DEF {
+namespace ufe {
+
+//! \brief Interface to create a UsdTransform3d interface object.
```
Comment (ppt-adsk) [inline lib/mayaUsd/ufe/UsdCameraHandler.h:26]: Copy-paste damage in comment.

File: lib/mayaUsd/ufe/UsdCameraHandler.h [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    // Delete the copy/move constructors assignment operators.
+    UsdCameraHandler(const UsdCameraHandler&) = delete;
+    UsdCameraHandler& operator=(const UsdCameraHandler&) = delete;
+    UsdCameraHandler(UsdCameraHandler&&) = delete;
+    UsdCameraHandler& operator=(UsdCameraHandler&&) = delete;
+
+    //! Create a UsdCameraHandler.
+    static UsdCameraHandler::Ptr create();
+
+    // Ufe::Transform3dHandler overrides
```
Comment (ppt-adsk) [inline lib/mayaUsd/ufe/UsdCameraHandler.h:44]: Copy-paste damage.

File: lib/mayaUsd/ufe/UsdCamera.h [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+    ~UsdCamera() override = default;
+
+    // Delete the copy/move constructors assignment operators.
+    UsdCamera(const UsdCamera&) = delete;
+    UsdCamera& operator=(const UsdCamera&) = delete;
+    UsdCamera(UsdCamera&&) = delete;
+    UsdCamera& operator=(UsdCamera&&) = delete;
+
+    //! Create a UsdCamera.
+    static UsdCamera::Ptr create();
```
Comment (ppt-adsk) [inline lib/mayaUsd/ufe/UsdCamera.h:48]: Do we need this and its associated setItem() call?  Seems like needless complexity otherwise.

**Review by ppt-adsk — COMMENTED** [review]
Looks good, only a few trivial changes to make.

I'd normally ask for automated tests to be included in the pull request, but in this case I understand that time is of the essence and I will assume that they will follow in a later pull request.

**Comment by williamkrick** [comment]
Logged MAYA-109133 to add automated tests.

**Review by ppt-adsk — APPROVED** [review]

**Comment by williamkrick** [comment]
Everything is protected by UFE version checks, so this one is safe to merge.

**Pull request merged**